docs/tutorials/blog.md

```diff
@@ -724,7 +729,7 @@ We're gonna get serious now. Let's build a form to create a new post in the our
 
 💿 Add a form to the new route
 
-```tsx filename=app/routes/admin/new.tsx lines=[1,5-24]
+```tsx filename=app/routes/admin/new.tsx lines=[1,4-25]
```

Comment:
If the whole `return` statement is highlighted it makes it easier for folks to copy/paste content (if they don't type it all themselves) as the previous version of the function only contained this body:

```
return <h2>New Post</h2>;
```